src/main/java/cc/hyperium/gui/ChangeBackgroundGui.java

```diff
@@ -55,23 +73,35 @@ public void updateScreen() {
     @Override
     protected void keyTyped(char typedChar, int keyCode) throws IOException {
         this.downloadUrlField.textboxKeyTyped(typedChar, keyCode);
-        if (keyCode == Keyboard.KEY_ESCAPE)
+        if (keyCode == Keyboard.KEY_ESCAPE) {
             Minecraft.getMinecraft().displayGuiScreen(this.prevGui);
-        if (keyCode == Keyboard.KEY_RETURN)
+            if(isFS) {
```

Comment:
This variable no longer exists?

Reply:
well it isn't needed now

Reply:
But you are still accessing it, thus causing an error since the variable does not exist.

Reply:
missed that one, my bad